tsconfig.json

```diff
@@ -0,0 +1,114 @@
+{
```

Comment:
The source code is written in JS. This file is only required (if it is.. as I am not sure) by the `test/types/...` - so if it must be in the repo, please move it there.

Also, please clean it up from the comments - the TS config file has its documentation, so there is no need to keep unused options.